slave-services/src/main/java/orq/iqu/slaveservices/dto/SourceDTO.java

```diff
@@ -1,79 +1,100 @@
 package orq.iqu.slaveservices.dto;
 
 public class SourceDTO {
-	private String id;
-	private String displayName;
-	private String description;
+  private String id;
+  private String displayName;
+  private String description;
+  private String image;
 
-	public SourceDTO() {
+  public SourceDTO() {
+  }
 
-	}
+  public SourceDTO(String id, String displayName, String description, String image) {
+    this.id = id;
+    this.displayName = displayName;
+    this.description = description;
+    this.image = image;
+  }
 
-	public SourceDTO(String id, String displayName, String description) {
-		this.id = id;
-		this.displayName = displayName;
-		this.description = description;
-	}
+  public String getImage() {
+    return image;
+  }
 
-	public String getId() {
-		return id;
-	}
+  public void setImage(String image) {
+    this.image = image;
+  }
 
-	public void setId(String id) {
-		this.id = id;
-	}
+  public String getId() {
+    return id;
+  }
 
-	public String getDisplayName() {
-		return displayName;
-	}
+  public void setId(String id) {
+    this.id = id;
+  }
 
-	public void setDisplayName(String displayName) {
-		this.displayName = displayName;
-	}
+  public String getDisplayName() {
+    return displayName;
+  }
 
-	public String getDescription() {
-		return description;
-	}
+  public void setDisplayName(String displayName) {
+    this.displayName = displayName;
+  }
 
-	public void setDescription(String description) {
-		this.description = description;
-	}
+  public String getDescription() {
+    return description;
+  }
 
-	@Override
-	public int hashCode() {
-		final int prime = 31;
-		int result = 1;
-		result = prime * result + ((description == null) ? 0 : description.hashCode());
-		result = prime * result + ((displayName == null) ? 0 : displayName.hashCode());
-		result = prime * result + ((id == null) ? 0 : id.hashCode());
-		return result;
-	}
+  public void setDescription(String description) {
+    this.description = description;
+  }
 
-	@Override
-	public boolean equals(Object obj) {
-		if (this == obj)
-			return true;
-		if (obj == null)
-			return false;
-		if (getClass() != obj.getClass())
-			return false;
-		SourceDTO other = (SourceDTO) obj;
-		if (description == null) {
-			if (other.description != null)
-				return false;
-		} else if (!description.equals(other.description))
-			return false;
-		if (displayName == null) {
-			if (other.displayName != null)
-				return false;
-		} else if (!displayName.equals(other.displayName))
-			return false;
-		if (id == null) {
-			if (other.id != null)
-				return false;
-		} else if (!id.equals(other.id))
-			return false;
-		return true;
-	}
+  @Override
+  public String toString() {
```

Comment:
Probably this should be removed, I don't see a reason for having toString in DTOs